user_guide_src/source/outgoing/view_cells.rst

```diff
@@ -249,7 +256,7 @@ You can pass additional parameters to the ``mount()`` method by passing them as
     }
 
     // Called in main View:
-    <?= view_cell('RecentPosts::show', ['categoryId' => 5]); ?>
+    <?= view_cell('RecentPosts', ['categoryId' => 5]) ?>
```

Comment:
This comment is regarding the `RecentPosts` cell classes above.

```php
// app/Cells/RecentPosts.php
namespace App\Cells;
use CodeIgniter\View\Cells\Cell;

class RecentPosts extends Cell
{
    protected $posts;
```

I think we should change it to:

```php
// app/Cells/RecentPostsCell.php
namespace App\Cells;
use CodeIgniter\View\Cells\Cell;

class RecentPostsCell extends Cell
{
    public $posts;
```
Our examples should follow the convention, and users should be able to copy-paste and run them.

Reply:
Fixed.